Gemfile

```diff
@@ -6,6 +6,8 @@ ruby '2.3.3'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
+gem 'minitest'
+gem 'minitest-vcr'
```

Comment:
Based on past experience I think it's probably best to use `minitest-around` rather than `minitest-vcr`. There are two reasons for this:

1. Using `minitest-around` allows you to use a single cassette across multiple `it` blocks. In this pull request there are currently four cassettes, but there are only two urls being hit so it only needs two cassettes.
2. `minitest-around` decouples your test names from your cassettes, so renaming a test doesn't cause your cassette to silently be re-recorded.

You can see `minitest-around` in action [in `wikisnakker`](https://github.com/everypolitician/wikisnakker/blob/d2bd31d9ac1ff10cf6498f0072523525d7b548ac/test/single_test.rb#L4).